Initial README.md (#1)
tjmelo/cx-experiences: node/README.md

Experiences by Framework Node.js
=

> > All of code snippets are experiences for application tests.

Reviewer: ```suggestion
> All the code snippets are experienced by tests in applications.
```
